### PR TITLE
Moving to use postgresql and adding basic docker

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Added docker file and compose .
Enabled flyway. 
Disabled Auto Schema Updates to manage with flyway.
Added v1 schema definition copying from camunda repos

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        - no_proxy
+    environment:
+      - CAMUNDA_DB_PASSWORD=camunda
+      - CAMUNDA_DB_HOST=host.docker.internal
+      - CLIENT_ID
+      - CLIENT_SECRET
+      - ISSUER_URI
+      - ENABLE_SCHEMA_UPDATE=true
+    ports:
+      - 8080:8080
```
Comment (timja) [inline docker-compose.yml]: can we not use port 8080 👅 
most standard port in the world and likely to cause clashes

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    depends_on:
+      - camunda-bpm-database
+  camunda-bpm-database:
+    build:
+      context: docker/database
+    ports:
+      - 5432:5432
```
Comment (timja) [inline docker-compose.yml]: same here

File: docker/database/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM postgres:9.6
```
Comment (timja) [inline docker/database/Dockerfile]: shouldn't we be using postgres 10 or 12?

File: build.gradle [hunk 4]
```diff
@@ -46,12 +60,28 @@ version = '1.0-SNAPSHOT'
 description = 'Camunda BPM Spring Boot Application'
 sourceCompatibility = '1.8'
 
-mainClassName = 'uk.gov.hmcts.reform.camunda.bmp.CamundaApplication'
+mainClassName = 'uk.gov.hmcts.reform.camunda.bpm.CamundaApplication'
```
Comment (timja) [inline build.gradle]: 🤦‍♂

File: docker/database/Dockerfile (hunk 3 above)
Comment (adusumillipraveen) [reply docker/database/Dockerfile]: I checked that we are not using latest in postgres tf module. Better to have same version as what we are using in there.

Comment (timja) [reply docker/database/Dockerfile]: I would prefer to bin that DB then, that was an accident, should be an easy fix

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    environment:
+      - CAMUNDA_DB_PASSWORD=camunda
+      - CAMUNDA_DB_HOST=host.docker.internal
+      - CAMUNDA_DB_PORT=5434
+      - CLIENT_ID
+      - CLIENT_SECRET
+      - ISSUER_URI
+      - ENABLE_SCHEMA_UPDATE=true
+    ports:
+      - 8082:8080
```
Comment (timja) [inline docker-compose.yml]: I think what you had clashes with one of the reform apps :p
```suggestion
      - 8999:8080
```

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      - ENABLE_SCHEMA_UPDATE=true
+    ports:
+      - 8999:8080
+    depends_on:
+      - camunda-bpm-database
+  camunda-bpm-database:
+    build:
+      context: docker/database
+    ports:
+      - 5434:5432
```
Comment (timja) [inline docker-compose.yml]:
```suggestion
      - 5898:5432
```

File: src/main/resources/application.yaml [hunk 7]
```diff
@@ -38,8 +34,10 @@ camunda:
...
       properties:
         generalResourceWhitelistPattern: '[.a-zA-Z0-9@_-]+'
+    database:
+      schema-update: ${ENABLE_SCHEMA_UPDATE:false}
 
 server:
   servlet:
     context-path: /
-  port: 8080
+  port: 8080
```
Comment (timja) [inline src/main/resources/application.yaml]: hmm wonder if we should change this?

**Pull request merged**